pkg/smartcontract/manifest/manifest.go

```diff
@@ -22,10 +22,12 @@ const (
 	NEP11StandardName = "NEP-11"
 	// NEP17StandardName represents the name of NEP-17 smartcontract standard.
 	NEP17StandardName = "NEP-17"
-	// NEP11Payable represents the name of contract interface which can receive NEP-11 tokens.
-	NEP11Payable = "NEP-11-Payable"
-	// NEP17Payable represents the name of contract interface which can receive NEP-17 tokens.
-	NEP17Payable = "NEP-17-Payable"
+	// NEP11Payable represents the name of NEP-26 smartcontract standard (contract interface
+	// which can receive NEP-11 tokens).
+	NEP11Payable = "NEP-26"
+	// NEP17Payable represents the name of NEP-27 smartcontract standard (contract interface
+	// which can receive NEP-17 tokens).
+	NEP17Payable = "NEP-27"
```

Comment:
I think we need a definition of NEP27StandardName as well. Then this one is just a compatibility alias.

Reply:
Added. Do you think we need to mark these values as deprecated? I did, but at the same time they are quite convenient, so may be we can keep both.